pkg/ccl/changefeedccl/testfeed_test.go

```diff
@@ -1318,10 +1318,42 @@ func (c *cloudFeed) appendParquetTestFeedMessages(
 	}
 	metaColumnNameSet := extractMetaColumns(columnNameSet)
 
+	// Count the number of distinct columns in the output so we can later verify
+	// that it matches the number of datums we output. Duplicating them in the
+	// output or missing one of these keys in the parquet output would be
+	// unexpected.
+	numDistinctColumns := 0
```

Comment:
nit: looks to me like you can just use len(seenColumnNames) instead of keeping track of the num yourself.

also nit: a set should be map[something]struct{}, not bool. unless you're actually using the bool which you dont appear to be doing here